stronghold/middleware.py

```diff
@@ -15,8 +16,10 @@ class LoginRequiredMiddleware(object):
 
     """
 
-    def __init__(self, *args, **kwargs):
+    def __init__(self, get_response=None, *args, **kwargs):
         self.public_view_urls = getattr(conf, 'STRONGHOLD_PUBLIC_URLS', ())
+        self.get_response = get_response
+        return super(LoginRequiredMiddleware, self).__init__(get_response, *args, **kwargs)
```

Comment:
Since there is no need to accept get_response as an argument, taking and passing \*args and \*\*kwargs should be sufficient.